CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased][unreleased]
 
+## [2.0.0] - 2018-02-15
+
+- Target .NET Standard 1.0.
+- Add documentation for method parameters.
```

Comment:
Can we put these changes under an "Added" header, like we've done for the 1.0.1 release.